addons/mail/static/src/discuss/call/common/meeting.js

```diff
@@ -3,54 +3,68 @@ import { Thread } from "@mail/core/common/thread";
 import { Call } from "@mail/discuss/call/common/call";
 import { CallActionList } from "@mail/discuss/call/common/call_action_list";
 import { ChannelInvitation } from "@mail/discuss/core/common/channel_invitation";
+import { useMessageScrolling } from "@mail/utils/common/hooks";
 
-import { Component, onMounted, onWillUnmount, useChildSubEnv, useRef, useState } from "@odoo/owl";
+import { Component, onMounted, onWillUnmount, useChildSubEnv, useSubEnv } from "@odoo/owl";
 
-import { isMobileOS } from "@web/core/browser/feature_detection";
-import { _t } from "@web/core/l10n/translation";
+import { Dropdown } from "@web/core/dropdown/dropdown";
 import { useService } from "@web/core/utils/hooks";
+import { MeetingSideActions } from "./meeting_side_actions";
+import { useThreadActions } from "@mail/core/common/thread_actions";
+import { useMessageSearch } from "@mail/core/common/message_search_hook";
 
 /** @typedef {"chat"|"invite"} MeetingPanel */
 
 /**
  * @typedef {Object} Props
- * @property {MeetingPanel} initialSidePanel
+ * @property {ThreadActionDefinition.id} [autoOpenAction]
  * @extends {Component<Props, Env>}
  */
 export class Meeting extends Component {
     static template = "mail.Meeting";
-    static props = ["initialSidePanel?"];
-    static components = { Call, CallActionList, ChannelInvitation, Composer, Thread };
+    static props = ["autoOpenAction?"];
+    static components = {
+        Call,
+        CallActionList,
+        ChannelInvitation,
+        Composer,
+        Dropdown,
+        MeetingSideActions,
+        Thread,
+    };
 
     setup() {
         this.store = useService("mail.store");
         this.ui = useService("ui");
         this.rtc = useService("discuss.rtc");
-        this.state = useState({
-            activeSidePanel: this.props.initialSidePanel,
-            jumpPresent: 0,
+        onMounted(() => {
+            if (this.props.autoOpenAction) {
+                this.threadActions.actions
+                    .find((a) => a.id === this.props.autoOpenAction)
+                    ?.onSelected();
+            }
+        });
```

Comment:
This is much more fragile than the state that was previously used. Component could mount for several reasons, changing the active action.